projects/components/src/table/table.component.ts

```diff
@@ -262,7 +261,7 @@ import { CsvFileName } from '../download-file/service/file-download.service';
       <div
         class="pagination-controls"
         *ngIf="this.pageable"
-        [style.position]="!this.showFloatingPaginator ? (this.isTableFullPage ? 'fixed' : 'sticky') : ''"
+        [style.position]="!this.showFloatingPaginator ? 'sticky' : ''"
```

Comment:
@aneeshsharma For a full-page table, after this change, does the paginator still stay at the same place when we scroll? Just want to make sure we are at parity with the existing use-case.

Reply:
Yep, with this change the paginator would go at the bottom of the table in case of a full page table and an embedded table.

If the height of the table is not enough to accommodate all the rows, we get scrolling inside the table as was before with `embedded` table, while the pagination element is fixed w.r.t the table. If the table height is enough, the pagination element stays at the bottom of the table.

Reply:
Okay, then we can't make this change. We need to support existing behavior. Could you please come up with another solution to solve the alignment problem while ensuring the Paginator is not scrolled away?